### PR TITLE
New portable generator icons

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Lexorion
imageadd: The portable PACMAN generators now have new icons, including on and off states.
/:cl:

Picture: top = old ones,middle = new ones (off) , bottom = new ones (on)

### CODE REVIEW

**Comment by Niknakflak** [comment]
It takes 5 seconds with file.house come on

**Comment by Lexorion** [comment]
I said I would do it,sorry NikNak editing this post in a second

**Comment by DoctoreMad** [comment]
TODO List: Make not 3/4

**Comment by fludd12** [comment]
These are insanely pretty holy frick

**Comment by WJohn** [comment]
Good shit lexorion.

**Comment by ChangelingRain** [comment]
The diamond pacman looks like its animation cycle is slightly off compared to the other two, though.

**Comment by RemieRichards** [comment]
Off in what way. slow? fast? It doesn't appear wrong.

**Comment by RemieRichards** [comment]
Oh hmm, the blue pixels in the center of each column last for 1 frame more than they should, you're right.

File: code/modules/power/port_gen.dm [hunk 1]
```diff
@@ -4,7 +4,8 @@
 	name = "portable generator"
 	desc = "A portable generator for emergency backup power."
 	icon = 'icons/obj/power.dmi'
-	icon_state = "portgen0"
+	icon_state = "portgen0_0"
+	var/base_icon = "portgen0"
```
Comment (Cyberboss) [inline code/modules/power/port_gen.dm]: new vars after parent vars

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Lexorion** [comment]
It's not easy to notice but looks like you are right. Mind if I do this in a different PR?

**Comment by Armhulen** [comment]
keep the old genny sprites for ancientstation

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**